Makefile

```diff
@@ -92,7 +92,14 @@ generate-modular-docs: generate-docs ## Generate modular command-line reference
 	SRC_DIR=$$(pwd)/docs/commands DEST_DIR=$$(pwd)/dist go run ./cmd/modular-docs
 .PHONY: generate-modular-docs
 
+lint-lang:
```

Comment:
Needs documentation to appear in the help command output (see others for example).